packages/components/scripts/vendorism/custom/textarea.vendorism.js

```diff
@@ -1,40 +1,92 @@
 import { removeSection } from '../remove-section.js';
+import { replaceSection, replaceSections } from '../replace-section.js';
+
+const FILES_TO_TRANSFORM = [
+  'textarea.component.ts',
+  'textarea.styles.ts',
+  'textarea.test.ts',
+];
+
+/**
+ * Transform the component code
+ * @param {String} path
+ * @param {String} originalContent
+ * @returns
+ */
+const transformComponent = (path, originalContent) => {
+  let content = removeSection(originalContent, '/** Draws a filled', 'filled = false;');
+  content = replaceSection(['filled', 'readonly'], content);
```

Comment:
This line can be removed. 
The string is replaced in line 21
```suggestion
```

Reply:
Leftover from refactoring :(